spyder/app/tests/test_mainwindow.py

```diff
@@ -6331,5 +6331,94 @@ def test_runfile_namespace(main_window, qtbot, tmpdir):
     assert "test_globals True" in control.toPlainText()
 
 
+@pytest.mark.skipif(
+    os.name == 'nt',
+    reason="No quotes in windows filepath"
```

Comment:
```suggestion
    reason="No quotes on Windows file paths"
```

Reply:
I don't have a windows machine handy so maybe a test could be added for this at a later point?